.husky/pre-push

```diff
@@ -1,11 +1,32 @@
 #!/usr/bin/env bash
 
-main_branch="canary"
+protected_branch="canary"
+protected_remote_url="git@github.com:vercel/next.js.git"
```

Comment:
my remote for vercel's next.js is `https://github.com/vercel/next.js.git` so maybe we need to check both?

Reply:
Good catch, updated it to check both